apps/api/src/app/controllers/search/search.controller.ts

```diff
@@ -51,4 +51,14 @@ export class SearchController {
     ): Promise<PaginateResult<ContentModel>> {
         return await this.searchService.searchContent(query, pageNum, contentFilter);
     }
+
+    @ApiTags(DragonfishTags.Search)
+    @Get('get-fandom-tag-content')
+    async getFandomTagContent(
+        @Query('tagId') tagId: string,
+        @Query('pageNum') pageNum: number,
+        @Cookies('contentFilter') contentFilter: ContentFilter
+    ): Promise<PaginateResult<ContentModel>> {
+        return await this.searchService.searchFandomTagContent(tagId, pageNum, contentFilter);
+    }
```

Comment:
Added method to get fandom tag content as part of search; in the future, this will be consolidated into a more advanced general search method, since #637 will require being able to search by tag anyway.

Reply:
nitpick: The name of this method confuses me a little--does it get the content of a fandom tag, or the set of all content _with_ a particular fandom tag? If the latter, maybe `getContentByFandomTag`?